python/paddle/tensor/math.py

```diff
@@ -562,34 +562,52 @@ def remainder(x, y, name=None):
 
 def multiply(x, y, axis=-1, name=None):
     """
-	:alias_main: paddle.multiply
-	:alias: paddle.multiply,paddle.tensor.multiply,paddle.tensor.math.multiply
+    multiply two tensors element-wise. The equation is:
 
-Examples:
+    .. math::
+        out = x * y
 
-    .. code-block:: python
+    **Note**:
+    ``paddle.multiply`` supports broadcasting. If you want know more about broadcasting, please refer to :ref:`user_guide_broadcasting` .
```

Comment:
In the future, you can write "if you would like to". "If you want to" is not a good English manner.